operator/pkg/ciliumenvoyconfig/annotations.go

```diff
@@ -16,9 +16,8 @@ import (
 )
 
 const (
-	servicePrefixAnnotation = annotation.Prefix + ".service"
-	lbEnabledAnnotation     = servicePrefixAnnotation + "/lb-l7"
-	lbModeAnnotation        = servicePrefixAnnotation + "/lb-l7-algorithm"
+	lbEnabledAnnotation = annotation.ServicePrefix + "/lb-l7"
+	lbModeAnnotation    = annotation.ServicePrefix + "/lb-l7-algorithm"
```

Comment:
I haven't kept the aliases here, since this functionality has not yet been documented. Let me know If it would be better to add them. Ref: #23100

Reply:
Cc @jrajahalme

Reply:
here is the docs for this feature (:disappear:)

https://github.com/cilium/cilium/pull/23100

Reply:
@sayboras Need to update the docs for the the uniform annotation style introduced here?